config/default/azwi.yaml

```diff
@@ -0,0 +1,253 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    azure-workload-identity.io/system: "true"
+  name: azure-wi-webhook-admin
+  namespace: system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  creationTimestamp: null
+  labels:
+    azure-workload-identity.io/system: "true"
+  name: azure-wi-webhook-manager-role
+  namespace: system
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  labels:
+    azure-workload-identity.io/system: "true"
+  name: azure-wi-webhook-manager-role
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - serviceaccounts
+    verbs:
+      - create
+      - get
+      - list
+      - patch
+      - update
+      - watch
```

Comment:
These permissions have been updated (`create`, `patch` and `update` are not required)

xref: https://github.com/Azure/azure-workload-identity/blob/release-1.0/deploy/azure-wi-webhook.yaml